AssemblyUtil.spec

```diff
@@ -25,6 +25,24 @@ module AssemblyUtil {
     funcdef get_assembly_as_fasta(GetAssemblyParams params) 
                 returns (FastaAssemblyFile file) authentication required;
 
+    /*
+           Structure for get_fasta function input and output:
+            rKBaseOjbReferences - is an array of KBase object references, such as KBaseGenomes.Genome, KBaseSets.AssemblySet, etc.
```

Comment:
Should explicitly list all the types this function handles and wrap line - KBase stnd is < 100 chars /line